streams/upgrade-system-tests-23/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Why do we need to write a new upgrade test?

Reply:
Yes. The upgrade system test needs compiled jars for different versions. Hence, we need to add a new module for each version (each module depends on a different---already released---Kafka Streams version).